build-github-branch

```diff
@@ -11,6 +11,7 @@ class BranchOnRepo {
 def branchesToBuild = [
     new BranchOnRepo('apache/james-project', 'master', '3.2.0-SNAPSHOT'),
     new BranchOnRepo('apache/james-project', '3.1.x', '3.1.0', '3.1.0'),
+    new BranchOnRepo('apache/james-project', '3.2.x', '3.2.0', '3.2.0'),
```

Comment:
There's a bug in previous 3.1.X branch, and also this one.
The version is the version declared in the pom file, so it should be:

`new BranchOnRepo('apache/james-project', '3.2.x', '3.2.1-SNAPSHOT', '3.2.0'),`

Reply:
BTW, I think we should release first before trying to make it works correctly. (we have to modify things in shell scripts...)